matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt

```diff
@@ -201,7 +201,11 @@ data class Event(
      */
     fun getDecryptedTextSummary(): String? {
         if (isRedacted()) return "Message Deleted"
-        val text = getDecryptedValue() ?: return null
+        val text = getDecryptedValue() ?: run {
```

Comment:
The fun `getDecryptedTextSummary` is weird, there is no i18n, I think I want to delete it...